.github/workflows/ci.cue

```diff
@@ -17,9 +18,11 @@ _versions: {
 	page_build?:   #Branches
 }
 
-#Action: "actions/checkout@v2"
+#Action: "actions/checkout@v2" | "actions/setup-beam@v1"
```

Comment:
This was left un-fixed, right? Or did you fix it further down the line?

Reply:
I just hacked the `ci.yaml` file directly to make it pass on this PR. The fixups came in a later patch once it was merged. That's what confuses you I think.